exporter/signalfxexporter/internal/translation/converter.go

```diff
@@ -134,17 +134,17 @@ func (c *MetricsConverter) metricToSfxDataPoints(metric pdata.Metric, extraDimen
 	return dps
 }
 
-func labelsToDimensions(labels pdata.StringMap, extraDims []*sfxpb.Dimension) []*sfxpb.Dimension {
-	dimensions := make([]*sfxpb.Dimension, len(extraDims), labels.Len()+len(extraDims))
+func attributesToDimensions(attributes pdata.AttributeMap, extraDims []*sfxpb.Dimension) []*sfxpb.Dimension {
+	dimensions := make([]*sfxpb.Dimension, len(extraDims), attributes.Len()+len(extraDims))
 	copy(dimensions, extraDims)
-	if labels.Len() == 0 {
+	if attributes.Len() == 0 {
 		return dimensions
 	}
-	dimensionsValue := make([]sfxpb.Dimension, labels.Len())
+	dimensionsValue := make([]sfxpb.Dimension, attributes.Len())
 	pos := 0
-	labels.Range(func(k string, v string) bool {
+	attributes.Range(func(k string, v pdata.AttributeValue) bool {
 		dimensionsValue[pos].Key = k
-		dimensionsValue[pos].Value = v
+		dimensionsValue[pos].Value = v.StringVal()
```

Comment:
Bit: add a link to https://github.com/open-telemetry/opentelemetry-collector/issues/3815 so that we can find it easily.

Reply:
I added it in the task list.